core/trino-main/src/test/java/io/trino/connector/AllowAllAccessControl.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.base.security;
+package io.trino.connector;
```

Comment:
It is is used only by tests now, by mock connector access control.

Reply:
what about renaming the class to eg TestingAllowAllAccessControl ?